src/deployment/azuredeploy.bicep

```diff
@@ -192,21 +179,6 @@ resource roleAssignments 'Microsoft.Authorization/roleAssignments@2020-10-01-pre
   ]
 }]
 
-// try to make role assignments to deploy as late as possible in order to have principalId ready
-resource roleAssignmentsNet 'Microsoft.Authorization/roleAssignments@2020-10-01-preview' = [for r in roleAssignmentsParams: {
-  name: guid('${resourceGroup().id}${r.suffix}-1f-net')
-  properties: {
-    roleDefinitionId: '/subscriptions/${subscription().subscriptionId}/providers/Microsoft.Authorization/roleDefinitions/${r.role}'
-    principalId: netFunction.outputs.principalId
-  }
-  dependsOn: [
-    eventGrid
-    keyVault
-    serverFarm
-    featureFlags
-  ]
-}]
-
 // try to make role assignments to deploy as late as possible in order to have principalId ready
```

Comment:
Did you mean to remove this comment?

Reply:
I removed the entire roleAssignment resource and the comment that preceded it, yes.